docs/source/using/troubleshooting.adoc

```diff
@@ -213,3 +213,16 @@ You can also use the GUI to add yourself to the Hyper-V Administrators group as
 Now you can run the Hyper-V commands as a normal user.
 
 For more options for Hyper-V see https://blogs.msdn.microsoft.com/virtual_pc_guy/2010/09/28/creating-a-hyper-v-administrators-local-group-through-powershell[creating Hyper-V administrators local group].
+
+== Hyper-V: Minishift running with Hyper-V fails when connected to OpenVPN
+If you try to use Minishift with Hyper-V using an external virtual switch, while you are connected to a VPN such as OpenVPN, Minishift fails to provision the VM and `minishift ssh` fails to connect.
+The following error is encountered:
+----
+Starting local OpenShift cluster using 'hyperv' hypervisor...
+ E0428 15:34:02.215336   10984 start.go:171] Error starting the VM: Error creating the VM. Error creating machine: Error detecting OS: Too many retries waiting for SSH to be available.  Last error: Maximum number of retries (60) exceeded. Retrying.
+ E0428 15:43:39.508106   10984 start.go:171] Error starting the VM: Error configuring authorization on host: Too many retries waiting for SSH to be available.  Last error: Maximum number of retries (60) exceeded.
+ Retrying.
+----
+Cause: When Minishift tries to provision the VM and reach the gateway, the packets are not routed over to the virtual switch, but end up in the VPN. You can verify this by using `minishift ip` which indicates that the instance has been assigned an IP address (say, for example, 10.0.21.126) and then pinging your gateway IP address using `minishift ssh ping 10.0.21.1`.
```

Comment:
Sorry for the 'out of context' replies. This is a side effect from using email to respond. A lot of things here, and I hope I have answered most in the conference call. But it might be a good thing to summarize some of the known limitations somewhere.